src/state/reducer.ts

```diff
@@ -13,18 +17,49 @@ import transactions from './transactions/reducer'
 import user from './user/reducer'
 import wallets from './wallets/reducer'
 
-export default {
-  application,
+const persistedReducers = {
   user,
   transactions,
   signatures,
+  lists,
+}
+
+const appReducer = combineReducers({
+  ...persistedReducers,
```

Comment:
This is error-prone because a duplicated property will overwrite it. Could you instead move `...persistedReducers` to the _bottom_ of the object definition?